callbacks/anstomlog.py

```diff
@@ -219,9 +219,9 @@ def v2_playbook_on_task_start(self, task, is_conditional):
     def _open_section(self, section_name):
         # pylint: disable=I0011,W0201
         self.tark_started = datetime.now()
-        self.task_start_preamble = "[%s.%03d] %s | " % (
-            self.tark_started.strftime("%Y-%m-%d %H:%M:%S"),
-            self.tark_started.microsecond / 1000, section_name)
+        self.task_start_preamble = "[%s] %s" % (
```

Comment:
Not completely sold to removing the ms from the timestamp, but if you feel this is better like this, let's go with that...